NEWS

```diff
@@ -6,7 +6,7 @@ New Code Generators
 
 * Add Ada generator from Stephane Carrez (#135).
 
-Javascript
+JavaScript
```

Comment:
Brand

Reply:
Not sure I really agree we should be kowtowing to Oracle here.

Reply:
Java and JavaScript aren't related beyond a branding coup selected by marketing folks at Netscape (which was later bought by AOL) who aligned LiveScript with the "up and coming" Java by Sun Microsystems (later bought by Oracle) making it JavaScript.

Reply:
Yes, I'm well aware of the history of this.

My dislike of the trademarked spelling here is (a) that it unhelpfully emphasises a connection to Java when really the two languages have very little in common (more than once I've had to try to resolve Java vs JS confusion by less technical management types) and (b) that it's now an Oracle-owned trademark which I don't really have any inclination to support.

Reply:
Frankly, you're inconsistent. The first item in here is `JavaScript`: https://github.com/snowballstem/snowball/blob/5b33b67cba5893571eabc060af16dd6440817d06/CONTRIBUTING.rst#key-problems-to-solve

If you wanted to make a political stand, you could use ECMAScript instead.

That said, this isn't meant to be a political PR, so I'll drop it.